lib/MetaCPAN/Document/Release.pm

```diff
@@ -1148,9 +1148,10 @@ sub modules {
     );
     return unless $ret->{hits}{total};
 
-    my @files = map { single_valued_arrayref_to_scalar($_) }
-        map +{ %{ $_->{fields} }, %{ $_->{_source} } },
-        @{ $ret->{hits}{hits} };
+    my @files = map {
+        single_valued_arrayref_to_scalar( $_->{fields} );
+        +{ %{ $_->{fields} }, %{ $_->{_source} } }
+    } @{ $ret->{hits}{hits} };
```

Comment:
ok, the first option works - let's go with this one.
I rather deal with single_valued_arrayref_to_scalar itself later (half its code isn't even used so we can simplify it).

I'll push a fix.